### ISSUE TITLE
Add new styleDarkContent

### ISSUE THREAD

**Issue by jcesarmobile** [apache/cordova-plugin-statusbar#164]
That sets the native value to `UIStatusBarStyleDarkContent`. Note that it's an iOS 13 value, so in older version it should set the value to `UIStatusBarStyleDefault` as it used to have the same effect (set text to black color)

**Comment by jcesarmobile** [comment]
You can with default, but it’s not released

**Comment by ShaneMosley** [comment]
So this fix will be implemented in the next release of this plugin? And if so, do you know when that might be?

**Comment by ShaneMosley** [comment]
Sorry, just a bit confused with your statement

**Comment by jcesarmobile** [comment]
Yes, will be on next release, or you can install from github directly

**Comment by ShaneMosley** [comment]
Oh legit! I see what you mean now. I can do it now with sticking with default because you already committed a change. I will do that :) Thanks again!

**Comment by exxbrain** [comment]
I think UIStatusBarStyleDefault will be useful as well for automatic theme changing.

**Comment by andresbelt** [comment]
for me works this 

if( this.style == 'dark') {
       this.statusBar.backgroundColorByHexString('#000000');
        this.statusBar.styleDefault()
       }else{
        this.statusBar.backgroundColorByHexString('#ffffff');
        this.statusBar.styleLightContent();
       }

But I change this line in CDVStatusBar.m in my xcode project 

 - (void) styleLightContent:(CDVInvokedUrlCommand*)command
{
    [self setStyleForStatusBar:UIStatusBarStyleDarkContent]; ---> this UIStatusBarStyleDarkContent for ios 13.0
}

**Comment by derweise** [comment]
> for me works this
> 
> if( this.style == 'dark') {
> this.statusBar.backgroundColorByHexString('#000000');
> this.statusBar.styleDefault()
> }else{
> this.statusBar.backgroundColorByHexString('#ffffff');
> this.statusBar.styleLightContent();
> }
> 
> But I change this line in CDVStatusBar.m in my xcode project
> 
> * (void) styleLightContent:(CDVInvokedUrlCommand*)command
>   {
>   [self setStyleForStatusBar:UIStatusBarStyleDarkContent]; ---> this UIStatusBarStyleDarkContent for ios 13.0
>   }

How can I retrieve in `CDVStatusBar.m` the iOS version of the device and then set the value accordingly?

**Comment by jtumain** [comment]
@derweise I did the following to achieve this. I made a new styleDarkContent function.

```
-(void) styleDarkContent:(CDVInvokedUrlCommand*)command
{
    if (@available(iOS 13, *)) {
        [self setStyleForStatusBar:UIStatusBarStyleDarkContent];
    } else {
        [self styleDefault:command];
    }
}
```

If iOS 13 or greater style dark content because it isn't supported by older versions; else use default.

I also added this to the header file.
`- (void) styleDarkContent:(CDVInvokedUrlCommand*)command;`

And if you're using Ionic or TS interfaces added a styleDarkContent function in the type definitions file.
